po/fi.po

```diff
@@ -1543,6 +1543,26 @@ msgstr "takaisin %d "
 msgid "pktlen must be within: %d < value <= %d"
 msgstr "pktlenin on oltava sisällä: %d < arvo <= %d"
 
+#: ping/ping_common.c:826 ping/ping_common.c:836
+#, c-format
+msgid " time=%ld.%03ld ms"
+msgstr " aika=%ld.%03ld ms"
+
+#: ping/ping_common.c:828
+#, c-format
+msgid " time=%ld ms"
+msgstr " aika=%ld ms"
+
+#: ping/ping_common.c:830
+#, c-format
+msgid " time=%ld.%01ld ms"
+msgstr " aika=%ld.%01ld ms"
+
+#: ping/ping_common.c:833
+#, c-format
+msgid " time=%ld.%02ld ms"
+msgstr " aika=%ld.%02ld ms"
+
```

Comment:
These changes try to revert recent change 98a6a4acc6f7f97f37a2729227b8e498da2edc10. Why?